docs/guide/Titanium_SDK/Titanium_SDK_Guide/Titanium_and_Angular/Titanium_Angular_Getting_Started_Guide.md

```diff
@@ -13,7 +13,7 @@ Titanium Angular projects are created using a new Angular project template. Open
 
 ```bash
 // appc-cli
-appc new -t app --ng
+ti create -t app --ng
```

Comment:
I'm not sure if this `--ng` flag was Appc CLI specific or not. @janvennemann what do you know?

Reply:
It's not in the Ti CLI

Reply:
This would now be `--template angular-default`.

```suggestion
ti create -t app --template angular-default
```